products/sle/main.pm

```diff
@@ -973,15 +977,6 @@ elsif (get_var('HPC')) {
         elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
```

Comment:
is really console/hostname needed for hpc/powerman? current powerman test-suite don't have HOSTNAME variable, so console/hostname sets default 'susetest' hostname, which is already set by installation

if it is not needed, it can be refactored as well